tensorflow_app/views/import_graphdef.py

```diff
@@ -244,8 +252,30 @@ def import_graph_def(request):
                         return JsonResponse({'result': 'error', 'error':
                                              'Missing shape info in GraphDef'})
 
+                if str(node.type) == 'MaxPool3D':
```

Comment:
Why do you need to explicitly apply `str` method? I saw it in a few places above -- `node.type` is already a string. I am a bit confused myself.